src/apps/search/SearchRoutes.tsx

```diff
@@ -16,35 +16,35 @@ const SearchRoutes = (props: Props) => (
   <Routes>
     <Route
       match='/events/'
-      element={<Event className={props.className} />}
+      element={<Event {...props} />}
```

Comment:
I think we can probably revert this change, in case we ever pass additional props to `SearchRoutes` that should not be passed along to the side panel components.